lib/css/components/badges.less

```diff
@@ -174,6 +183,44 @@
     .highcontrast-mode({ border-color: currentColor; });
 }
 
+//  $$  States
+//  ---------------------------------------------------------------------------
+&.s-badge__danger {
+    color: var(--red-900);
+    background-color: var(--red-100);
+    border-color: var(--red-600);
+
+    &.s-badge__filled {
+        color: @white;
+        background-color: var(--red-500);
+        border-color: transparent;
+    }
+}
+
+&.s-badge__info {
+    color: var(--blue-900);
+    background-color: var(--blue-100);
+    border-color: var(--blue-600);
+}
+
+&.s-badge__warn {
```

Comment:
Let's match our other alias class names and use `warning` here

Reply:
D'oh! Nice catch!